include/geos/operation/buffer/BufferOp.h

```diff
@@ -119,6 +119,8 @@ class GEOS_DLL BufferOp {
 
     geom::Geometry* resultGeometry;
 
+    bool isInvertOrientation;
```

Comment:
I tend to prefer inline initialization for the relief of mind rather than doing it in each constructor:

```suggestion
    bool isInvertOrientation = false;
```